### ISSUE TITLE
Output problem with ONNX inference

### ISSUE THREAD

**Issue by Ranclus** [derronqi/yolov7-face#80]
The output tensor is tensor: float32[1,25200,21]. What is the meaning of the 21 values for each detection? I can't find the information, and without that, I can't use the model. Pls

**Comment by iperov** [comment]
traditionally, Yolo repositories have the dirtiest code

**Comment by Ranclus** [comment]
Yes (Although not only the yolo ones. But the IA ones). I'm not even asking about the code. Just how to use it.

**Comment by iperov** [comment]
are you serious? how are you going to use it without understanding the code?

**Comment by Ranclus** [comment]
Why would I need to actually understand this particular code JUST to use it? You only need to know what's the input, and what's the output. Everything else is unnecesary for the sole purpose of using the model. For now, I know whats the input, so, I do the inference. But I don't know how to interpret the output.

I am surprised that this information isn't always the first thing displayed on the main page of the repositories.

**Comment by iperov** [comment]
lmao ok

**Comment by wangsouza** [comment]
> The output tensor is tensor: float32[1,25200,21]. What is the meaning of the 21 values for each detection? I can't find the information, and without that, I can't use the model. Pls

I have the same problem.
I think 21 corresponds to: 1 channel, 4 bounding box, 1 score, 15 landmarks

**Comment by potapov-dm** [comment]
5 for bbox (x_min, y_min, x_max, y_max, confidence), 1 for class, 15 for landmarks (x_1, y_1, landmark_confidence_1, ...). Not sure about order.
